metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/EcloudBasePluginParameters.java

```diff
@@ -0,0 +1,50 @@
+package eu.europeana.metis.core.workflow.plugins;
+
+/**
+ * Class that contains basic parameters required for each {@link eu.europeana.cloud.service.dps.DpsTask}
+ * that is sent to ECloud.
+ *
+ * @author Simon Tzanakis (Simon.Tzanakis@europeana.eu)
+ * @since 2019-03-19
+ */
+public class EcloudBasePluginParameters {
+
+  private final String ecloudBaseUrl;
+  private final String ecloudProvider;
+  private final String ecloudDataset;
```

Comment:
Call it ecloudDatasetId? It is an id number after all.

Reply:
Fixed.